Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h

```diff
@@ -40,12 +40,18 @@
 #include <string.h>
 #endif
 #if defined(__cplusplus)
-#if __has_include(<ptrauth.h>)
+#if defined(__arm64e__) && __has_include(<ptrauth.h>)
```

Comment:
What version of Xcode did you use? We still make releases with 14.3 in CI, I guess this was made locally with 15.x, but it should be fine.